app/BlinkLab/BlinkLab.py

```diff
@@ -133,74 +141,91 @@ def run(self):
     #======================== private ==========================================
     
     def runExperimentForSize(self,networksize):
-        # log
-        printAndLog('NEWNETWORKSIZE', {'networksize': networksize})
-        
-        # configure the manager's ACL
-        mote_list = [TAG_EUI]+[ALLMOTES[a] for a in range(networksize)]
-        for m in mote_list:
-            
+        try:
```

Comment:
it looks to me like the biggest change here is that you put the code in a try/except. Why?